stdlib/2and3/timeit.pyi

```diff
@@ -26,8 +26,10 @@ if sys.version_info >= (3, 5):
                number: int = ..., globals: Optional[Dict[str, Any]] = ...) -> float: ...
     def repeat(stmt: _stmt = ..., setup: _stmt = ..., timer: _Timer = ...,
                repeat: int = ..., number: int = ..., globals: Optional[Dict[str, Any]] = ...) -> List[float]: ...
+    def main(args: Optional[Sequence[str]]) -> None: ...
 else:
     def timeit(stmt: _stmt = ..., setup: _stmt = ..., timer: _Timer = ...,
                number: int = ...) -> float: ...
     def repeat(stmt: _stmt = ..., setup: _stmt = ..., timer: _Timer = ...,
                repeat: int = ..., number: int = ...) -> List[float]: ...
+    def main(args: Optional[Sequence[str]]) -> None: ...
```

Comment:
Since both branches are identical, you should de-indent this instance and remove the first one.